### PR TITLE
Add fortran-binary Python package

### PR DESCRIPTION
**Pull request by robertodr**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [ ] When in trouble be please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by robertodr** [comment]
@vahtras do you agree to be maintainer of this package?

**Review by dopplershift — COMMENTED** [review]
Pending maintainer confirmation.

**Comment by vahtras** [comment]
> Pending maintainer confirmation.

I confirm (if his is the way to do it...)

**Comment by robertodr** [comment]
@conda-forge/help-python this is ready to go

**Comment by robertodr** [comment]
@conda-forge/help-python @dopplershift anything else I should do to move this forward?

**Review by dopplershift — APPROVED** [review]

File: recipes/fortran-binary/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "fortran-binary" %}
+{% set version = "1.0.6" %}
+{% set sha256 = "be613276fb3141b81933c508990b8fa5e71f1df04a7423567d2aa78a16267241" %}
```
Comment (dopplershift) [inline recipes/fortran-binary/meta.yaml:3]: Can you actually undo the templating of the sha and build? It's not usually done any more like that and can make things more difficult for the auto-update bot.

**Review by dopplershift — CHANGES_REQUESTED** [review]

**Review by dopplershift — APPROVED** [review]

**Pull request merged**